python/cugraph/cugraph/dask/link_analysis/pagerank.py

```diff
@@ -105,8 +157,20 @@ def pagerank(input_graph,
         numerical roundoff. Usually values between 0.01 and 0.00001 are
         acceptable.
 
-    nstart : not supported
-        initial guess for pagerank
+    nstart : cudf.Dataframe, optional (default=None)
+        GPU Dataframe containing the initial guess for pagerank.
+        nstart['vertex'] : cudf.Series
+            Subset of vertices of graph for initial guess for pagerank values
+        nstart['values'] : cudf.Series
+            Pagerank values for vertices
+
+    weight: str, optional (default=None)
+        The attribute column to be used as edge weights if Graph is a NetworkX
+        Graph. This parameter is here for NetworkX compatibility and is ignored
+        in case of a cugraph.Graph
+
+    dangling : dict, optional (default=None)
+        This parameter is here for NetworkX compatibility and ignored
 
     Returns
     -------
```

Comment:
We should probably also include the following as part of the description of the return value:

`NOTE: if the input cugraph.Graph was created using the renumber=False option of any of the from_*_edgelist() methods, pagerank assumes that the vertices in the edgelist are contiguous and start from 0. If the actual set of vertices in the edgelist is not contiguous (has gaps) or does not start from zero, pagerank will assume the "missing" vertices are isolated vertices in the graph, and will compute and return pagerank values for each.`